azure/template.json

```diff
@@ -86,7 +86,8 @@
         "ProviderCommitmentsSiteUrl": "",
         "SignProviderAgreementUrl": "",
         "ReservationsSiteUrl": "",
-        "EmployerFavouritesUrl": ""
+        "EmployerFavouritesUrl": "",
+		"TrainingProviderPermissionUrl": ""
```

Comment:
dodgy formatting

Reply:
i know , looks okay in VS code.. i have used jsonformatter  , thats why loads of diff.